File: apis/backend/src/main/java/com/tms/api/objects/viewmodels/organization/OrganizationProjectViewModel.java
package com.tms.api.objects.viewmodels.organization;

import java.util.List;

import com.tms.api.objects.models.organization.OrganizationModel;
import com.tms.api.objects.models.project.ProjectModel;

public record OrganizationProjectViewModel(
    OrganizationModel organization,
    List<ProjectModel> projects
) {}

File: ticket-managment-system-web/src/app/core/view-models/organization/organization-project.view-model.ts
import { OrganizationModel, ProjectModel } from "../../models";

export class OrganizationProjectViewModel {
    public organization!: OrganizationModel;
    public projects: ProjectModel[] = [];

    constructor(organization: Partial<OrganizationProjectViewModel> = {}) {
        Object.assign(this, organization);
    }
}